src/Analyzers/Core/Analyzers/AbstractBuiltInCodeStyleDiagnosticAnalyzer_Core.cs

```diff
@@ -40,7 +39,7 @@ protected AbstractBuiltInCodeStyleDiagnosticAnalyzer(ImmutableArray<DiagnosticDe
             Debug.Assert(!supportedDiagnostics.Any(descriptor => descriptor.CustomTags.Any(t => t == WellKnownDiagnosticTags.Unnecessary)) || this is AbstractBuiltInUnnecessaryCodeStyleDiagnosticAnalyzer);
         }
 
-        public virtual CodeActionRequestPriority RequestPriority => CodeActionRequestPriority.Normal;
+        public virtual bool IsHighPriority => false;
```

Comment:
this is the internal IBuiltInAnalyzer api.  it currently only ever needs to know if something is high pri or not.  It exists at a layer that can't reference Workspaces.  So it was much simpler to just make this a boolean.